sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/StorageTest.java

```diff
@@ -144,28 +186,36 @@ public void readEndToEnd() throws Exception {
         Storage storage = Storage.getInstance();
         Channel channel = start(storage);
 
-        AppCenterFuture<Document<TestDocument>> doc = Storage.read(PARTITION, DOCUMENT_ID);
+        AppCenterFuture<Document<TestDocument>> doc = Storage.read(PARTITION, DOCUMENT_ID, TestDocument.class);
```

Comment:
I am talking about test cases like this... need to have the test setup... /**setup **/ /** Call **/ /**verify */

Reply:
Keep in mind that `/**` with 2 stars is only at class, method or field level (Javadoc).
Regular code comments is just `/*`

Reply:
I decided to move set up to `setUp` method in the parent class. So the tests are less cluttered now. It seems like it's a bit more readable and these comments may not be as necessary as they were before.